src/componants/Articles.jsx

```diff
@@ -36,8 +37,15 @@ function Articles() {
 			setArticles(results);
 			setLoaded(true);
       setSearchParams(searchObject)
-		});
-	}, [topic, sortBy, order]); 
+		})
+    .catch((error) => {
+      if(error){
+        setError('Page not found')
+      }
```

Comment:
Same with this one. As a user, "Page not found" sounds like a generic message. You could use the error message from your API which should give something like `Topic not found`